2
runtime error: returning outside function